tests/test_tspr_cases_integration.py

```diff
@@ -0,0 +1,207 @@
+import datetime
+import sys
+import os
+import unittest
+import subprocess
+import re
+from pathlib import Path
+
+sys.path.append("./constrain")
+
+
+class TestTSPRCasesIntegration(unittest.TestCase):
```

Comment:
Integration of the demo case to make sure that it keeps working with subsequent updates of ConStrain.